packages/flutter/test/material/paginated_data_table_test.dart

```diff
@@ -786,4 +786,34 @@ void main() {
     await tester.pumpWidget(buildTable(false));
     expect(find.byType(Checkbox), findsNothing);
   });
+
+  testWidgets('Table should not use decoration from DataTableTheme', (WidgetTester tester) async {
+    final Size originalSize = binding.renderView.size;
```

Comment:
Is setSurfaceSize necessary for this test?

Reply:
Yes, it's seems like the PaginatedDataTable cannot be rendered without it as it will overflow if the screen is too small. It's the same for most of the tests in this file.